presto-hive/pom.xml

```diff
@@ -244,6 +224,26 @@
 
         <!-- for testing -->
         <dependency>
+            <groupId>org.apache.curator</groupId>
+            <artifactId>curator-test</artifactId>
+            <version>3.3.0</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
```

Comment:
I think we should just remove this zookeeper dependency for testing in order to resolve the dependency conflict. Since the ``zkclient`` already include the required zookeeper version and both they are just set in test scope. It would be much nicer if that works.